Sources/Brave/Frontend/Widgets/LoadingViewController.swift

```diff
@@ -37,6 +37,15 @@ public class LoadingViewController: UIViewController {
 public class AuthenticationController: LoadingViewController {
   enum AuthViewType {
     case sync, tabTray
+    
+    var description: String {
```

Comment:
I would rename this to something else.
`description` is pretty much discouraged everywhere: https://developer.apple.com/documentation/swift/string/description

Same for dictionary and so on.

Reply:
Fixed at https://github.com/brave/brave-ios/pull/7878/commits/2873a2ad927225f02157498543c1e57e183f24ee